Makefile

```diff
@@ -18,72 +17,71 @@ REVIEWDOG_REPO=github.com/haya14busa/reviewdog/cmd/reviewdog
 # Also it builds the docs and the generators
 .PHONY: testsuite
 testsuite:
-	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) testsuite || exit 1;)
-	#$(MAKE) -C generator test
+	@$(foreach var,$(PROJECTS),$(MAKE) -C $(var) testsuite || exit 1;)
 
 stop-environments:
-	$(foreach var,$(PROJECTS_ENV),$(MAKE) -C $(var) stop-environment || exit 0;)
+	@$(foreach var,$(PROJECTS_ENV),$(MAKE) -C $(var) stop-environment || exit 0;)
 
 # Runs unit and system tests without coverage and race detection.
 .PHONY: test
 test:
-	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) test || exit 1;)
+	@$(foreach var,$(PROJECTS),$(MAKE) -C $(var) test || exit 1;)
 
 # Runs unit tests without coverage and race detection.
 .PHONY: unit
 unit:
-	$(foreach var,$(PROJECTS),$(MAKE) -C $(var) unit || exit 1;)
+	@$(foreach var,$(PROJECTS),$(MAKE) -C $(var) unit || exit 1;)
 
 .PHONY: coverage-report
 coverage-report:
-	mkdir -p $(COVERAGE_DIR)
-	# Writes atomic mode on top of file
-	echo 'mode: atomic' > ./$(COVERAGE_DIR)/full.cov
-	# Collects all coverage files and skips top line with mode
-	-tail -q -n +2 ./filebeat/$(COVERAGE_DIR)/*.cov >> ./$(COVERAGE_DIR)/full.cov
-	-tail -q -n +2 ./packetbeat/$(COVERAGE_DIR)/*.cov >> ./$(COVERAGE_DIR)/full.cov
-	-tail -q -n +2 ./winlogbeat/$(COVERAGE_DIR)/*.cov >> ./$(COVERAGE_DIR)/full.cov
-	-tail -q -n +2 ./libbeat/$(COVERAGE_DIR)/*.cov >> ./$(COVERAGE_DIR)/full.cov
-	go tool cover -html=./$(COVERAGE_DIR)/full.cov -o $(COVERAGE_DIR)/full.html
+	@mkdir -p $(COVERAGE_DIR)
+	@echo 'mode: atomic' > ./$(COVERAGE_DIR)/full.cov
+	@# Collects all coverage files and skips top line with mode
+	@-tail -q -n +2 ./filebeat/$(COVERAGE_DIR)/*.cov >> ./$(COVERAGE_DIR)/full.cov
```

Comment:
Seems like we miss heartbeat? :-) Perhaps better to also use a for loop here (if possible)?